File: software/examples/python/example_simple.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-  

HOST = "localhost"
PORT = 4223
UID = "XYZ" # Change to your UID

from tinkerforge.ip_connection import IPConnection
from tinkerforge.bricklet_analog_out import AnalogOut

if __name__ == "__main__":
    ipcon = IPConnection() # Create IP connection
    ao = AnalogOut(UID, ipcon) # Create device object

    ipcon.connect(HOST, PORT) # Connect to brickd
    # Don't use device before ipcon is connected

    # Set output voltage to 3.3V
    ao.set_voltage(3300)

    raw_input('Press key to exit\n') # Use input() in Python 3

File: software/examples/php/ExampleSimple.php
<?php

require_once('Tinkerforge/IPConnection.php');
require_once('Tinkerforge/BrickletAnalogOut.php');

use Tinkerforge\IPConnection;
use Tinkerforge\BrickletAnalogOut;

$host = 'localhost';
$port = 4223;
$uid = '7MJ'; // Change to your UID

$ipcon = new IPConnection(); // Create IP connection
$ao = new BrickletAnalogOut($uid, $ipcon); // Create device object

$ipcon->connect($host, $port); // Connect to brickd
// Don't use device before ipcon is connected

// Set output voltage to 3.3V
$ao->setVoltage(3300);

echo "Press key to exit\n";
fgetc(fopen('php://stdin', 'r'));

?>
